server/http.go

```diff
@@ -42,42 +42,56 @@ func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err := r.ParseForm(); err != nil {
-		http.Error(w, "Bad request body", http.StatusBadRequest)
+	if r.Header.Get("Content-Type") != "application/json" {
+		res := fmt.Sprintf("Expected Content-Type 'application/json' for webhook request, received '%s'.", r.Header.Get("Content-Type"))
+		http.Error(w, res, http.StatusBadRequest)
 		return
 	}
 
-	var webhook zoom.Webhook
-	decoder := schema.NewDecoder()
-
-	// Try to decode to standard webhook
-	if err := decoder.Decode(&webhook, r.PostForm); err != nil {
+	b, err := ioutil.ReadAll(r.Body)
+	if err != nil {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
 	}
 
-	p.handleStandardWebhook(w, r, &webhook)
+	var webhook zoom.Webhook
+	err = json.Unmarshal(b, &webhook)
+	if err != nil {
+		p.API.LogError("Error unmarshaling webhook", "err", err.Error())
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
 
-	// TODO: handle recording webhook
-}
+	if webhook.Event != zoom.EventTypeMeetingEnded {
+		// TODO: handle recording webhook
```

Comment:
Maybe open a ticket for meeting recording and remove the comment?

Reply:
https://github.com/mattermost/mattermost-plugin-zoom/issues/110 comment removed :+1: